scripts/emerge/resolve_drs_uris.py

```diff
@@ -0,0 +1,73 @@
+"""This script resolves drs uris to gs paths."""
+
+import argparse
+import json
+from pydoc import resolve
+import requests
+from pprint import pprint
+
+from oauth2client.client import GoogleCredentials
+
+
+def get_access_token():
+    """Get access token."""
+
+    scopes = ["https://www.googleapis.com/auth/userinfo.profile", "https://www.googleapis.com/auth/userinfo.email"]
+    credentials = GoogleCredentials.get_application_default()
+    credentials = credentials.create_scoped(scopes)
+
+    return credentials.get_access_token().access_token
+
+
+def write_list_to_file(input_list, outfile_name):
+    """Write a list of values to a file."""
+
+    with open(outfile_name, "w") as outfile:
+        for item in input_list:
+            outfile.write(f"{item}\n")
+
+
+def resolve_drs_uri(drs_uuid):
+    """Resolve a drs uri to return the corresponding gs path."""
+
+    print(f"resolving {drs_uuid}")
+    uri = f"https://data.terra.bio/ga4gh/drs/v1/objects/{drs_uuid}"
+    print(uri)
+
+    headers = {"Authorization": "Bearer " + get_access_token(), "accept": "*/*"}
+
+    response = requests.get(uri, headers=headers)
+    status_code = response.status_code
+
+    if status_code != 200:
+        return response.text
+
+    gs_path = response.json()["access_methods"][0]["access_url"]["url"]
```

Comment:
does `access_methods` include a list of items? how do we know we only want the first one? what if the first one isn't a google path?